### PR TITLE
Added invert method to ordinal scale

### PR DESCRIPTION
**Pull request by jordanmontt**
If we see the implementation of the NSOrdinalScale, we see that the scale method `NSOrdinalScale>>#scale:` creates a linear scale to scale the numbers. See:

```st
^ NSScale linear
		domain: {domain first. domain last};
		range: {range first. range last};
		scale: x
```

So it is scaling with a linear scale. This PR add the equivalent invert: method that inverts using the same linear scale. Maybe it is not the best solution but it is consistent with the current implementation.

### CODE REVIEW

File: src/NumericInterpolator/NSOrdinalScale.class.st [hunk 1]
```diff
@@ -39,7 +39,15 @@ NSOrdinalScale >> initialize [
...
-	self shouldNotImplement
+
+	^ NSScale linear
+		  domain: {
+				  domain first.
+				  domain last };
+		  range: {
+				  range first.
+				  range last };
+		  invert: anObject
```
Comment (PalumboN) [inline src/NumericInterpolator/NSOrdinalScale.class.st]: But this is not the same implementation as `scale:`? Should not range and domains be inverted here?

**Review by PalumboN — COMMENTED** [review]
Can we add tests? We already had problems with the scales because they are not tested :(

I also have questions about the expects of the current change that the tests would clarify.

**Comment by akevalion** [comment]
Yes more tests for `NSOrdinalScale >> #invert:`

File: src/NumericInterpolator/NSOrdinalScale.class.st (hunk 1 above)
Comment (akevalion) [reply src/NumericInterpolator/NSOrdinalScale.class.st]: in this case he is using `invert:`

Comment (jordanmontt) [reply src/NumericInterpolator/NSOrdinalScale.class.st]: we are creating a linear scale with the limits of the domain and range and then we return the invert the number. It is not the same as the `scale:`, there they scale the method.

**Comment by jordanmontt** [comment]
Yes I will add more tests :)

File: src/NumericInterpolator/NSOrdinalScale.class.st (hunk 1 above)
Comment (PalumboN) [reply src/NumericInterpolator/NSOrdinalScale.class.st]: Ohhh yes!!! Stupid rasta!

**Pull request merged**